src/guides/anatomy-of-an-http-transaction.md

```diff
@@ -0,0 +1,360 @@
+    // do whatever we need to in order to respond to this request.
+  });
+}).listen(80); // Activates this server, listening on port 80.
+```
+
+So far we haven't touched on the `response` object at all, which is an instance
+of [`ServerResponse`][]. It contains many useful methods for sending data back
+to the client. We'll cover that next.
+
+## HTTP Status Code
+
+If you don't bother setting it, the HTTP status code on a response will always
+be 200 by default. Of course, not every HTTP response warrants this, and at some
+point you'll definitely want to send a different status code. To do that, you
+can set the `statusCode` property.
+
+```javascript
+response.statusCode = 404; // Tell the client that the resource wasn't found.
+```
+
+There are some other shortcuts to this, as we'll see soon.
+
+## Response Headers
+
+Headers are set through a convenient method called [`setHeader`][].
+
+```javascript
+response.setHeader('Content-Type', 'application/json');
+response.setHeader('X-Powered-By', 'bacon');
+```
+
+When setting the headers on a response, the case is insensitive on their names.
+
+## Explicitly Send Header Data
+
+The methods of setting the headers and status code that we've already discussed
+assume that you're using "implicit headers". This means you're counting on node
+to send the headers for you at the correct time before you start sending body
+data.
+
+If you want, you can *explicitly* write the headers to the response stream.
+(Remember, the `response` object is a [`WriteableStream`][].) To do this,
+there's a method called [`writeHead`][], which writes the status code and the
+headers to the stream.
+
+```javascript
+response.writeHead(200, {
+  'Content-Type': 'application/json',
+  'X-Powered-By': 'bacon'
+});
+```
+
+Once you've set the headers (either implicitly or explicitly), you're ready to
+start sending response data).
+
+## Sending Response Body
+
+Since the `response` object is a [WriteableStream][], writing a response body
+out to the client is just a matter of using the usual stream methods.
+
+```javascript
+response.write('<html>');
+response.write('<body>');
+response.write('<h1>Hello, World!</h1>');
+response.write('</body>');
+response.write('</html>');
+response.end();
+```
+
+Remember, the `end` function on streams can also take in some optional data to
+send as the last bit of data on the stream, so we can simplify the example above
+as follows.
+
+```javascript
+response.end('<html><body><h1>Hello, World!</h1></body></html>');
+```
+
+> **Note:** It's important to set the status and headers *before* you start
+writing chunks of data to the body. This makes sense, since headers come before
+the body in HTTP responses.
+
+## Another quick thing about Errors
+
+The `response` stream can also emit `'error'` events, and at some point you're
+going to have to deal with that as well. All of the advice or `request` stream
+errors still applies here.
+
+## Put it all together
```

Comment:
I had left that out for brevity.

Initially, after seeing your comment here, my thoughts were "I'll just add a note mentioning that, and warning users to do as I say, not as I do", but then I remembered how much copying and pasting is likely to happen, so I'll add something in.